packages/rum-recorder/src/domain/rrweb/types.ts

```diff
@@ -1,5 +1,5 @@
 import { IdNodeMap, INode, SerializedNodeWithId } from '../rrweb-snapshot/types'
-import type { FocusRecord, RawRecord } from '../../types'
+import { FocusRecord, RawRecord } from '../../types'
```

Comment:
Did the compatibility tests break on this one?
if not, should they?

Reply:
Yes it has. I'm unsure why it didn't break on master. I am assuming that `domain/rrweb/types.d.ts` was not imported by `index.d.ts`, but now it (indirectly).

Reply:
oh yeah the addition of exports from `rumRecorderPublicApi` is indeed pulling some extra stuff.
it makes sense, it seems to still be a reasonable way to perform this compatibility check then.